i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,45 @@
+from torch import nn
+
+
+class ConformerFeedForwardv1(nn.Module):
```

Comment:
In our earlier implementation in RETURNN-common, we decided to call this `ConformerPositionwiseFeedForward`. Why this name change?

Reply:
The name used here is decided in the last implementation meeting

Reply:
So why was it decided that way? Why the name change?

Reply:
I think most of the people including me are not aware of the name ConformerPositionwiseFeedForward is used in RETURNN-common. If we want to use the consistent name as in RETURNN-common, I can change the name here

Reply:
@mmz33 was working on that, and then me as well. We had lots of earlier discussions on all of this, like naming of modules, etc. So I wonder a bit why we ignore all of that now.

Reply:
For reference: https://github.com/rwth-i6/returnn_common/pull/58

Reply:
Thanks for the reference, I have now also made activation configurable as in RETURNN-common

Reply:
I would have expected that we stick to our earlier discussions and decisions, and not redo all of those discussions, which are all not really so important anyway.

And introducing inconsistencies to our other existing implementation for no good reason is also not nice.

Reply:
@albertz in earlier discussions and decisions nearly no one took part. We specifically started over having as much people involved as possible. I definitely see it as a problem that you can not join the Tuesday meeting, and we should find a solution for that, e.g. have another date.

Also if the discussions are not important, then I do not understand the large number of comments from you here.

Reply:
> Also if the discussions are not important, then I do not understand the large number of comments from you here.

It should not be inconsistent to the existing implementation.

Also, we already had some experience with the earlier decisions, and some things were changed later on. So now re-deciding that again first takes takes time, and then also ignores that earlier experience.

So, I would stick to the original decisions, and be consistent to the existing implementation, unless there are really good reasons to change some parts.